external-crates/move/move-compiler/src/shared/mod.rs

```diff
@@ -179,6 +195,21 @@ pub struct IndexedPackagePath {
 
 pub type AttributeDeriver = dyn Fn(&mut CompilationEnv, &mut ModuleDefinition);
 
+#[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord)]
+pub struct KnownFilterKey {
+    name: String,
+    attribute_name: E::AttributeName_,
+}
+
+impl KnownFilterKey {
+    pub fn new(name: String, attribute_name: E::AttributeName_) -> Self {
```

Comment:
```suggestion
    pub fn new(name: impl Into<String>, attribute_name: E::AttributeName_) -> Self {
```
or
```suggestion
    pub fn new(name: impl ToString, attribute_name: E::AttributeName_) -> Self {
```
would help clean up the call sites a bit

Reply:
Cool trick! But in this case using `&str` should suffice as well to clean up the call sites